docker/Dockerfile

```diff
@@ -39,35 +39,22 @@ RUN rm -f /tmp/apache-maven-3.3.9.tar.gz
 
 # get node
```

Comment:
This is a suggestion for all of the Dockerfile. We might as well go to Ubuntu 24:
`FROM ubuntu:24.04

ENV user newman
ENV group newman
ENV uid 1001
ENV gid 1001

RUN apt-get update -y
RUN apt-get install coreutils wget -y
`